src/darwin/Framework/CHIP/MTRCommissionableBrowser.h

```diff
@@ -20,12 +20,22 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @protocol MTRCommissionableBrowserDelegate;
+@class MTRDeviceController;
 
 @interface MTRCommissionableBrowser : NSObject
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)new NS_UNAVAILABLE;
-- (instancetype)initWithDelegate:(id<MTRCommissionableBrowserDelegate>)delegate queue:(dispatch_queue_t)queue;
+- (instancetype)initWithDelegate:(id<MTRCommissionableBrowserDelegate>)delegate
+                      controller:(MTRDeviceController *)controller
+                           queue:(dispatch_queue_t)queue;
+/**
+ * Start browsing the available networks (e.g IP, BLE) for commissionable nodes.
```

Comment:
What happens if start happens twice without stop in between?  Will that fail?  No-op?  Something else?